Source/SvgElement.cs

```diff
@@ -346,6 +349,7 @@ protected internal virtual bool PushTransforms(ISvgRenderer renderer)
         protected internal virtual void PopTransforms(ISvgRenderer renderer)
         {
             renderer.Transform = _graphicsTransform;
+            _graphicsTransform.Dispose();

```

Comment:
Is it certain that `_graphicsTransform` cannot be `null` at this point?

Reply:
@mrbean-bremen 

Thanks for your review.
Immediately after this, it is set to null.
Is the answer correct ?

Reply:
No, I mean can it be already `null` at this point? That would crash if trying to dispose it. I checked the code, but couldn't exclude that.

Reply:
@mrbean-bremen 

Previous step:
```cs
renderer.Transform = _graphicsTransform;
```
This setting does not allow `null`, never _graphicsTransform becomes `null`.

Reply:
Sorry, still don't get it - if `_graphicsTransform` is `null` before that (which it is initially, and also after calling this method), `null` would be assigned to the Transform matrix, and the next line would crash. So my question is: is it ensured that `_graphicsTransform` cannot be `null` when entering this method?

Reply:
@mrbean-bremen

PushTransforms and PopTransforms are pair.So, be called PopTransforms after PushTransforms.

ref. #537

Reply:
Yes, I understand that, but as this is not enforced... anyway, you are right - calling it otherwise would be a programming error and would be caught in the review or by the crash.
Generally, I'm used to put this kind of symmetric actions into a context, at least in Python and C++ (which I'm using far more than C#, so bear with me...), but this seems to be less common in C#. 
Anyway, I'm merging now - thanks again for the contribution!

Reply:
@mrbean-bremen 

I decided that this method should remain as it is because this method is `protected internal`, this time.